doc/source/_templates/layout.html

```diff
@@ -2,38 +2,16 @@
 
 {%- block extrahead %}
 <style>
-  .main {
-    -moz-box-shadow: none;
-    -webkit-box-shadow: none;
-    box-shadow: none;
-  }
-  div.top-scipy-org-logo-header {
-    background-color: #fafafa;
-    border-bottom: 2px solid #013243; /* Warm Black */
-    margin-top: 0;
-    box-shadow: none;
-  }
-  div.top-scipy-org-logo-header img {
-    height: 100px;
-    padding-left: 27px;
-  }
-  div.spc-navbar .nav-pills > li > a {
-    background-color: #4d77cf; /* Han Blue */
-  }
+.navbar-brand img {
+   height: 75px;
+}
+.navbar-brand {
+   height: 75px;
+}
 </style>
 {{ super() }}
 {% endblock %}
 
-{%- block header %}
-<div class="container">
-  <div class="top-scipy-org-logo-header">
-    <a href="{{ pathto('index') }}">
-      <img border=0 alt="NumPy" src="{{ pathto('_static/numpylogo.svg', 1) }}">
-    </a>
-  </div>
-</div>
-
-{% endblock %}
 {% block rootrellink %}
```

Comment:
I'm not sure this block,  or the one below it does anything with the new theme - it looks like those block names are specific to the scipy template.

Reply:
(meaning we should probably just remove them)

Reply:
https://www.sphinx-doc.org/en/master/templating.html suggests block rootrellink is standard. It may still be unused here though.

Reply:
Maybe we could get one of the pydata-sphinx-theme people to weight in? @jorisvandenbossche  or @choldgraf could you take a look at what we can remove from `layout.html` and in general give us some pointers?

Reply:
`rootrellink` is indeed a standard block of sphinx, but in sphinx's default layout, that is used in the `relbar1` ("relation bar"), which is a block that we don't use in the pydata-sphinx-theme. 

So indeed, this can be removed. 
Also the sidebarsearch below is not used

Reply:
cleaned up in 699654a